### DISCUSSION TITLE
想许愿标签二级分类功能，感谢

### DISCUSSION THREAD

**Discussion by pupupuxi** [WatsonYao/nian_app#133] [Q&A] [answered]
我创建了几十个标签，虽然可以进行排序，但是在标签管理页面看着还是有点乱乱的，也不好找。希望可以增加一下标签检索的功能，和标签二级分类的功能。比如我创建了运动，学习，娱乐的小标签→可以归纳到生活这个大标签里。然后点击大标签，可以选择展开或者收回去小标签，比较方便整理。

**Comment by WatsonYao** [comment, marked as answer]
nian不支持标签层级，单层级标签限制下，推荐使用标签组合，可以备注多个标签。
也可以在标签管理页面自己添加emoji或其他格式代表大的分类，方便定位子标签。
